assets/stylesheets/commons/_header.css

```diff
@@ -0,0 +1,101 @@
+.header_bg{
+	background: rgba(124,178,40,0.9);
+}
+
+.wrap{
```

Comment:
Should have space between class name with {.
E.g: `.wrap {` instead of `.wrap{`